lib/adjust/index.js

```diff
@@ -0,0 +1,7 @@
+import curryN from '../curryN'
+
+export default curryN(3, function adgust(fn, i, arr) {
+  if (i >= arr.length || i < 0) return arr
```

Comment:
I would replace it with different condition to not repeat return arr twice, it saves around 10B I suppose

Reply:
I think we should stay with arr[i] instead of this condition

```js
const arr = [0, 1, 2];
console.log(arr[-1]) // undefined
console.log(arr[3]) // undefined
```
just check !!arr[i]